assets/airports/egcc.json

```diff
@@ -1,890 +1,3153 @@
-                "23R": ["_05L", "TABLY", "HON58", "SANBA"]
+                "EGCC23R": []
+            },
+            "body": ["_MCT231002", ["_MCT184007", "A30+|A50-"]],
+            "exitPoints": {
+                "LISTO": [["LISTO", "A50"]]
             },
-            "draw": [["_05R", "TABLY", "HON58", "SANBA"]]
+            "draw": [
+                ["_MCT2313.2", "_MCT184007", "LISTO"]
+            ]
         },
-
-        "SONEX": {
-            "icao": "SONEX",
-            "name": "SONEX",
-            "suffix": {"23L":"1Y", "23R":"1R"},
+        "MONTY1S": {
+            "icao": "MONTY1S",
+            "name": "Monty One Sierra",
+            "rwy": {
+                "EGCC05L": []
+            },
+            "body": ["^_MCT051002", ["_MCT334006", "A25+|A50-"], "XOBRO", ["_WAL081016", "A50"], "^_WAL081002", "_WAL184005"],
+            "exitPoints": {
+                "MONTY": ["MONTY"]
+            },
+            "draw": [
+                ["_MCT051002", "_MCT334006", "XOBRO", "_WAL081016", "_WAL081002", "_WAL184005", "MONTY"]
+            ]
+        },
+        "MONTY1Z": {
+            "icao": "MONTY1Z",
+            "name": "Monty One Zulu",
+            "rwy": {
+                "EGCC05R": []
+            },
+            "body": ["^_MCT051002", ["_MCT334006", "A25+|A50-"], "XOBRO", ["_WAL081016", "A50"], "^_WAL081002", "_WAL184005"],
+            "exitPoints": {
+                "MONTY": ["MONTY"]
+            },
+            "draw": [
+                ["_MCT051002", "_MCT334006", "XOBRO", "_WAL081016", "_WAL081002", "_WAL184005", "MONTY"]
+            ]
+        },
+        "MONTY1Y": {
+            "icao": "MONTY1Y",
+            "name": "Monty One Yankee",
+            "rwy": {
+                "EGCC23L": []
+            },
+            "body": ["_MCT2313.2", ["_MCT254008", "A25+|A50-"], ["_MCT254013", "A40+|A50-"], "^_MCT254015", ["_POL219036", "A50"]],
+            "exitPoints": {
+                "MONTY": ["MONTY"]
+            },
+            "draw": [
+                ["_MCT2313.2", "_MCT254008", "_MCT254013", "_MCT254015", "_POL219036", "MONTY"]
+            ]
+        },
+        "MONTY1R": {
+            "icao": "MONTY1R",
+            "name": "Monty One Romeo",
+            "rwy": {
+                "EGCC23R": []
+            },
+            "body": ["_MCT231003", ["_MCT254008", "A25+|A50-"], ["_MCT254013", "A40+|A50-"], "^_MCT254015", ["_POL219036", "A50"]],
+            "exitPoints": {
+                "MONTY": ["MONTY"]
+            },
+            "draw": [
+                ["_MCT231003", "_MCT254008", "_MCT254013", "_MCT254015", "_POL219036", "MONTY"]
+            ]
+        },
+        "POL4S": {
+            "icao": "POL4S",
+            "name": "Pole Hill Four Sierra",
+            "rwy": {
+                "EGCC05L": []
+            },
+            "body": ["_RWY23R", "_MCT053002", "_MCT053007", ["_POL182012", "A40+|A50-"], ["_POL182008", "A50"]],
```

Comment:
I'm going to stop adding comments on the `A50-` restrictions because they're everywhere and I'm pretty sure you get it by now and don't need me to point out each line that should be changed. Same goes for the ordering of the procedures.